web/src/components/ConditionalText/ConditionalText.tsx

```diff
@@ -0,0 +1,27 @@
+import { Typography, Theme } from "@mui/material";
+import { FC } from "react";
+import { useTheme } from "@mui/material/styles";
+
+interface Props {
+  isPositive: boolean;
```

Comment:
What `positive` means?

Reply:
On the end that label is going to be red od blue. Attached screenshot explains everything.
Would it be better to change it to 'isLabelBlue'? Or maybe you have something better in mind?

<img width="445" alt="Screenshot 2022-05-04 at 11 48 04" src="https://user-images.githubusercontent.com/41304693/166659473-bab33140-e548-43c5-82e4-86338a251790.png">

Reply:
I would remove this component and set logic directly, as this component is very simple and is used in 2 places only near to each other. We should stick to DRY often, but making separate component just to take care about such minor changes is overkill IMO

![Screenshot 2022-05-10 at 14 21 49](https://user-images.githubusercontent.com/96073847/167627084-0324fcc3-7481-4005-b9f3-c765b8a6a6c4.png)